features/user_can_see_menu.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Menus displayed for visitors
+  As a visitor
+  In order to help me decide what food to order
+  I would like to see Menus displayed
+
+  Background:
+    Given the following restaurants exists
+      | name            | description                                      |
+      | ThaiTanic       | Thailands finest food, watch out for the iceberg |
+      | Fu King Chinese | Best chinese food in town                        |
+    And the following menus exist for "ThaiTanic"
+      | name    |
```

Comment:
You should change name to menu_name just for clarification.

Reply:
Ok, will do

Reply:
Not necessary to change that to `menu_name` we're in the `Menu` domain so `name` is assumed to be the name of the menu